### DISCUSSION TITLE
Vs code error pathinclude using c++

### DISCUSSION THREAD

**Discussion by Sismo2003** [microsoft/vscode-cpptools#10093] [Q&A]
This morning I woke up and noticed that my Mac got updated to macOS Ventura. I was trying to do my homework and now I have this problem. I’m trying to fix it, but I can’t. If someone knows how to fix it, please help me, lmao.
<img width="773" alt="Screenshot 2022-10-31 at 19 17 15" src="https://user-images.githubusercontent.com/116782426/199137792-cc468f4b-ea55-4852-8077-82e75ae820b5.png">
<img width="1158" alt="Screenshot 2022-10-31 at 19 18 31" src="https://user-images.githubusercontent.com/116782426/199137899-0590cb7e-9529-4588-adce-6a7d53f90aa1.png">
<img width="1050" alt="Screenshot 2022-10-31 at 19 19 21" src="https://user-images.githubusercontent.com/116782426/199137959-f22f146b-af59-4aad-8561-9be0f7332adf.png">

**Comment by sean-mcmanus** [comment]
There's some problem with /usr/bin/clang++ on your machine so it's using /usr/bin/clang instead. Are you able to compile with your build tools? Are you able to see which compiler is being used?

**Reply by Sismo2003** [reply]
Thank you I change the path and it works.

**Comment by sem1hs** [comment]
did you solve problem bro ?

**Reply by sem1hs** [reply]
because i have same problem and i must solve

**Comment by Sismo2003** [comment]
> did you solve problem bro ?

Yes problem solved by editing the PATH for the compiler
